configuration/forms.md

```diff
@@ -556,3 +556,18 @@ Note that `FORM_CODE` should be replaced with the form code as defined in the JS
 
 ### Automatic changes with XLSForms
 If using a [XLSForm](http://xlsform.org/) and using [`medic-conf`](https://github.com/medic/medic-conf) to convert to XForm, the necessary fields will be automatically added to the resulting XForm. You can override the default prefix and separator by declaring `sms_keyword` and `sms_seperator` respectively in the Settings tab.
+
+## Supported XLSForm Meta Fields
+The following meta elements are supported:
+| element | type | description | value |
```

Comment:
The table markdown isn't being applied properly when I view the file in github. Adding a newline between lines 561 and 562 fixes it. I can commit that change if you'd like.

Reply:
Please do.